servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/server/subsystems/Karaf2xPublishController.java

```diff
@@ -213,12 +209,20 @@ public int transferBuiltModule(IModule[] module, IPath srcFile, IProgressMonitor
 
 	@Override
 	public int removeModule(IModule[] module, IProgressMonitor monitor) throws CoreException {
-		BundleDetails bd = new ModuleBundleVersionUtility().getBundleDetails(module, null);  // This will clearly break... wtf.  Crap. 
+		IPath tmpArchive = getTempBundlePath(module);
+		
+		BundleDetails bd = new ModuleBundleVersionUtility().getBundleDetails(module, tmpArchive);  // This will clearly break... wtf.  Crap. 
```

Comment:
I suppose that it is already supported by the ModuleBundleVersionutility. It depends on the Module type deployed if we look at the implentation here: https://github.com/fusesource/fuseide/blob/master/servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/server/subsystems/publish/ModuleBundleVersionUtility.java#L45